.NET/Microsoft.Recognizers.Definitions.Common/Portuguese/DateTimeDefinitions.cs

```diff
@@ -703,5 +703,9 @@ public static class DateTimeDefinitions
       public const string QuarterTypeRegex = @"(trimestral(mente)?)$";
       public const string SemiAnnualTypeRegex = @"(semestral(mente)?)$";
       public const string YearTypeRegex = @"(anual(mente)?)$";
+      public static readonly IList<string> FutureTerms = new List<string>
+        {
+            @"esea"

```

Comment:
There are two issues here. One, this term is not a "future" term, so we need a better name. 
Second, this value is incorrect in PT, it should be "esse", "essa", "este", "esta".